library/std/src/sys/windows/stdio.rs

```diff
@@ -169,13 +170,14 @@ fn write(
 }
 
 fn write_valid_utf8_to_console(handle: c::HANDLE, utf8: &str) -> io::Result<usize> {
-    let mut utf16 = [0u16; MAX_BUFFER_SIZE / 2];
+    let mut utf16: [MaybeUninit<u16>; MAX_BUFFER_SIZE / 2] = MaybeUninit::uninit_array();
```

Comment:
You might want to replace uses of `uninit_array`. See #101179.

Reply:
Oh! Just noticed you commented there with some disagreement. I'll defer to your preference here as I don't really have one.

Reply:
Not a huge fan but fair enough.

Reply:
Nah I'll change it. I don't feel that strongly and don't feel like causing issues for someone during rollup.